libraries/AP_Math/AP_Math.h

```diff
@@ -34,15 +34,15 @@ auto const constrain_int32 = &constrain_value<int32_t>;
  * @brief: Gets two values and returns the smaller one.
  */
 template<typename A, typename B>
-auto MIN(const A one, const B two) -> decltype(one < two ? one : two) { 
+auto min(const A one, const B two) -> decltype(one < two ? one : two) { 
```

Comment:
this clashes with libc and as such should be avoided. I don't see a point in changing this again. In the end it doesn't matter if it's a function or a macro... just leave it as is.